test/trans.js

```diff
@@ -0,0 +1,129 @@
+var fs = require("fs");
+var assert = require("assert");
+var {Context} = require('../lib/context');
+var {Environment} = require('../lib/environment');
+var {Template} = require("../lib/template");
+var {TemplateSyntaxError} = require('../lib/errors');
+var ilib = require("../../ilib");
```

Comment:
This path works for tests and in ringo on a web server, but only when ilib is checked out right next to reinhardt. In ringo running on jetty, you can use simply require("ilib") and it will find it properly in the packages dir, but that doesn't work for the tests. Is there some way to add to the path where the test framework looks for packages?